### PR TITLE
telco5g: Run nightly jobs on BOS2 upstream lab

### PR DESCRIPTION
**Pull request by sshnaidm**

### CODE REVIEW

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-cnftests

File: ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh [hunk 1]
```diff
@@ -49,13 +52,24 @@ EOF
...
-  ADDITIONAL_ARG="--cluster-name ${PREPARED_CLUSTER[0]} --force" 
-  if [[ "$T5CI_VERSION" == "4.14" ]]; then 
-    ADDITIONAL_ARG="--cluster-name ${PREPARED_CLUSTER[1]} --force"
-  fi 
+    ADDITIONAL_ARG="--cluster-name ${PREPARED_CLUSTER[0]} --force"
+    if [[ "$T5CI_VERSION" == "4.14" ]]; then
+        ADDITIONAL_ARG="--cluster-name ${PREPARED_CLUSTER[1]} --force"
+    fi
 else
   ADDITIONAL_ARG="-e $CL_SEARCH --exclude ${PREPARED_CLUSTER[0]} --exclude ${PREPARED_CLUSTER[1]}"
```
Comment (just-another-dude) [inline ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh:60]: Modify indent to 4 spaces for congruency

Comment (mgourin) [inline ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh:60]: Modify indent to contain 4 spaces for congruence.

File: ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh [hunk 2]
```diff
@@ -49,13 +52,24 @@ EOF
 ADDITIONAL_ARG=""
 # default to the first cluster in the array, unless 4.14
```
Comment (mgourin) [inline ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh:53]: "# default to the first cluster in the array, unless 4.14"
- what is the reasoning behind this?

Comment (sshnaidm) [reply ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh:53]: https://issues.redhat.com/browse/CNF-7848

File: ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh (hunk 1 above)
Comment (sshnaidm) [reply ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh:60]: it IS 4 spaces now

Comment (sshnaidm) [reply ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh]: done

**Pull request merged**